tutorial/create-a-simple-squid.md

```diff
@@ -445,7 +447,7 @@ export class SworkWorksReportSuccessEvent {
 
 After having obtained wrappers for Events and the metadata changes across different Runtime versions, it's finally time to define Handlers for these Events and attach them to our [Processor](../key-concepts/processor.md), and this is done in the `src/processor.ts` file in the project folder.
 
-First of all, we need to import the generated Entity model classes, in order to be able to use them in our code. And then, we need the type definitions of Crust events, so that they can be used to wrap them. So let's add these two lines at the top of our file:
+First of all, we need to import the generated Entity model classes, in order to be able to use them in our code. And then, we need the type definitions of Crust events, so that they can be used to wrap them. So let's add these two lines at the top TODO of our file:
```

Comment:
It's not clear should we delete the content of already existing `processor.ts` or just add to the top. 

But in case deleting the whole content of the file and step by step adding required code blocks then we will have incorrect file. In my case, I decided just to take the result file suggested at the end of this step. But the file itself has typos like `extrinisicId` that I renamed to `extrinsicId`.

I think that we should have proper code blocks that will be put to the empty file step by step. As a result, we will have the file without any errors.

Reply:
At the current moment we have a bug: after running processor and GraphQL

```
Prometheus metrics are served at port 3000
QueryFailedError: invalid byte sequence for encoding "UTF8": 0x00
    at QueryFailedError.TypeORMError [as constructor] (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/typeorm/error/TypeORMError.js:9:28)
    at new QueryFailedError (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/typeorm/error/QueryFailedError.js:13:28)
    at PostgresQueryRunner.<anonymous> (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/typeorm/driver/postgres/PostgresQueryRunner.js:281:31)
    at step (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/tslib/tslib.js:143:27)
    at Object.throw (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/tslib/tslib.js:124:57)
    at rejected (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/tslib/tslib.js:115:69)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  query: 'SELECT "Account"."id" AS "Account_id" FROM "account" "Account" WHERE "Account"."id" = $1 LIMIT 1',
  parameters: [ '\b�i.\x17Ɠ�\fU~�⡺*&��;�6k��>oC\x00x' ],
  driverError: error: invalid byte sequence for encoding "UTF8": 0x00
      at Parser.parseErrorMessage (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/pg-protocol/dist/parser.js:287:98)
      at Parser.handlePacket (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/pg-protocol/dist/parser.js:126:29)
      at Parser.parse (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/pg-protocol/dist/parser.js:39:38)
      at Socket.<anonymous> (/Users/dmitrylavrenov/subsquid/squid-template/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:527:28)
      at addChunk (node:internal/streams/readable:324:12)
      at readableAddChunk (node:internal/streams/readable:297:9)
      at Readable.push (node:internal/streams/readable:234:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 115,
    severity: 'ERROR',
    code: '22021',
    detail: undefined,
    hint: undefined,
    position: undefined,
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'wchar.c',
    line: '2061',
    routine: 'report_invalid_encoding'
  },
  length: 115,
  severity: 'ERROR',
  code: '22021',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'wchar.c',
  line: '2061',
  routine: 'report_invalid_encoding'
}
```